app/main/views/frameworks.py

```diff
@@ -233,6 +234,29 @@ def framework_updates_email_clarification_question():
                 e, current_user.supplier_id, current_user.email_address))
         abort(503, "Clarification question email failed to send")
 
+    # Send confirmation email to the user who submitted the question
+    # No need to fail if this email does not send
+    email_body = render_template(
+        "emails/clarification_question_submitted.html",
+        user_name=current_user.name,
+        message=clarification_question
+    )
+    try:
+        send_email(
+            current_user.email_address,
+            email_body,
+            current_app.config['DM_MANDRILL_API_KEY'],
+            current_app.config['CLARIFICATION_EMAIL_SUBJECT'],
+            current_app.config['CLARIFICATION_EMAIL_FROM'],
+            current_app.config['CLARIFICATION_EMAIL_NAME'],
+            ["clarification-question-confirm"]
```

Comment:
Are these worth storing in config variables?  The last one I put in a config variable because the email was to be a secret.  We're not planning on overriding this one, are we?